mkdocs.yml

```diff
@@ -9,6 +9,7 @@ pages:
     - Packages: application_packages.md
     - Operators: operator_development.md
     - AutoMetric API: autometrics.md
+    - Custom Control Tuple Support: control_tuple_support.md
```

Comment:
Custom Control Tuple Support => Custom Control Tuples
{better to keep these navigation panel strings short}